scipy/stats/tests/test_continuous.py

```diff
@@ -686,9 +713,9 @@ def has_formula(order, kind):
     for i in range(3, 6):
         ref = dist.moment(i, 'central', method='quadrature')
         check(i, 'central', 'normalize', ref,
-              success=has_formula(i, 'standardized'))
+              success=has_formula(i, 'standardized') and not np.any(variance == 0))
```

Comment:
I excluded the 'normalize' method for central moments in the degenerate case where the variance is zero, and that needs to be reflected here.

Reply:
Do we still need this? Hard to tell after the force push.

Reply:
Yes, this is still needed for now. Tests will fail otherwise. You had mentioned avoiding this by keeping degenerate cases out of the typical parameter range. They already are excluded though, so I suppose this would amount to making tests for discrete distributions. I think there's a need for a principled solution that could be left for a follow-up, and hope this is fine as a temporary convenience. 

In any case, I think the tests could use some significant cleaning up, which I think will require removing hypothesis, and it may be better to keep this hacked in skip for now, and not try to mess with testing too much before the big rewrite of test code.

Reply:
> You had mentioned avoiding this by keeping degenerate cases out of the typical parameter range. They already are excluded though...

What happened is that they get injected because they occur at endpoints of the domains of parameters ($n=0$, $p=0$, $p=1$).